share/github-backup-utils/ghe-backup-es-hookshot

```diff
@@ -20,21 +20,33 @@ ghe_remote_version_required "$host"
 # Make sure root backup dir exists if this is the first run
 mkdir -p "$GHE_SNAPSHOT_DIR/hookshot"
 
-indices=$(ghe-ssh "$host" 'curl -s "localhost:9201/_cat/indices/hookshot-logs-*"' | cut -d ' ' -f 3)
-
-current_index=hookshot-logs-$(ghe-ssh "$host" 'date +"%Y-%m-%d"')
-
-for index in $indices; do
-  if [ -f $GHE_DATA_DIR/current/hookshot/$index.gz -a -f $GHE_DATA_DIR/current/hookshot/$index.gz.complete -a $index \< $current_index ]; then
-    # Hard link any older indices that are complete, since these won't change
-    ln $GHE_DATA_DIR/current/hookshot/$index.gz $GHE_SNAPSHOT_DIR/hookshot/$index.gz
-    ln $GHE_DATA_DIR/current/hookshot/$index.gz.complete $GHE_SNAPSHOT_DIR/hookshot/$index.gz.complete
+if [ $GHE_VERSION_MAJOR -ge 2 ] && [ $GHE_VERSION_MINOR -ge 2 ]; then
+  es_port=9201
+else
+  es_port=9200
+fi
```

Comment:
We weren't making this differentiation before, and won't need it soon. Can we remove this and put 👇 back to the way it was before.

Reply:
I added this in for a couple of reasons:

- To keep it consistent with the audit log script.
- As I now catch errors when retrieving the hookshot log indices, we need to fall back to 9200 in case we are running on GHE 2.0 or 2.1, which doesn't have Elasticsearch available on 9201. An alternative would be to not run `ghe-backup-es-hookshot` on these older versions since hookshot logs aren't in Elasticsearch there, but this would involve adding additional version logic to `bin/ghe-backup`.

Reply:
👍 make sense in the current context.